packages/gpl-core/src/post.ts

```diff
@@ -77,13 +79,14 @@ export class Post {
   }
 
   public async update(
-    newMetadataUri: String,
+    newMetadataUri: string,
     postAccount: anchor.web3.PublicKey,
     profileAccount: anchor.web3.PublicKey,
     userAccount: anchor.web3.PublicKey,
-    owner: anchor.web3.PublicKey) {
+    owner: anchor.web3.PublicKey,
+    sessionTokenAccount: anchor.web3.PublicKey | null = null) {
```

Comment:
Can you use the sessionTokenAccount? instead of manually defaulting to null. This will make it easy for typedoc, IDEs to pick up the nature of the arguments easily.

Reply:
This is fixed!